src/lib/bcmp/bcmp_cli.cpp

```diff
@@ -8,6 +8,10 @@
 #include "bcmp.h"
 #include "bcmp_cli.h"
 
+#include "configuration.h"
+
+using namespace cfg;
```

Comment:
since bcmp.h no longer includes these the cli needs them for now so it can still work